test/libethereum/blockchain.cpp

```diff
@@ -98,7 +107,7 @@ void doBlockchainTests(json_spirit::mValue& _v, bool _fillin)
 
 		// construct true blockchain
 		TransientDirectory td;
-		FullBlockChain<Ethash> trueBc(rlpGenesisBlock.out(), StateDefinition(), td.path(), WithExisting::Kill);
+		FullBlockChain<Ethash> trueBc(rlpGenesisBlock.out(), AccountMap(), td.path(), WithExisting::Kill);
 
 		if (_fillin)
```

Comment:
A lot of the code down from here is commented out for now. This is only for the creation of BlockChainTests, not for actually running them. This is rather complex and will get fixed later.